src/WinRT.Interop.Generator/References/InteropReferences.cs

```diff
@@ -1651,7 +1661,7 @@ public InteropReferences(
             returnType: _corLibTypeFactory.Void,
             parameterTypes: [
                 _corLibTypeFactory.UInt32,
-                ABIType.MakePointerType()]));
+                AbiType.MakePointerType()]));
```

Comment:
Do you need ToValueTypeSignature here?

Reply:
Mmh this code was the same before, but yeah doesn't hurt to fix it, thank you! 🙂